tests/e2e/test_rosacli_account_roles.go

```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
 			}
 		})
 })
+
+var _ = Describe("Create account roles", labels.Feature.AccountRoles, func() {
+	defer GinkgoRecover()
+
+	var (
+		rosaClient                     *rosacli.Client
+		ocmResourceService             rosacli.OCMResourceService
+		accountRolePrefixesNeedCleanup = make([]string, 0)
```

Comment:
You do not need to create an empty array. It should be correctly handled by `len` and `append` methods
```suggestion
		accountRolePrefixesNeedCleanup []string
```

Reply:
update according to comment.